spec/1_base.adoc

```diff
@@ -65,11 +65,10 @@ A GeoPackage SHALL have the file extension name ".gpkg".
 [caption=""]
 .Requirement 4
 ====
-A GeoPackage SHALL only contain data elements, SQL constructs and GeoPackage extensions with the "gpkg" author name specified in this encoding standard.
+A GeoPackage SHALL only contain the data elements (tables, columns, or values) and SQL constructs  (views, data types, indexes, constraints, or triggers) specified in the core of this encoding standard (<<features>>, <<tiles>>, and <<attributes>>). Extended GeoPackages MAY contain additional data elements and SQL constructs as specified through the <<extension_mechanism>>.
```

Comment:
Yep, and this is one reason I haven't merged this PR yet. We are having a hard time coming up with language that distinguishes safe things from potentially unsafe things. Presumably an index is safe but it is possible to make a dangerous index too. Hopefully we will wrap this one up next week.